label_studio/tasks/models.py

```diff
@@ -992,6 +992,50 @@ class Meta:
         db_table = 'prediction'
 
 
+class FailedPrediction(models.Model):
+    """
+    Class for storing failed prediction(s) for a task
+    """
+    message = models.TextField(
+        _('message'),
+        default='',
+        blank=True,
```

Comment:
I figure we shouldn't allow this to be blank, right? Is there a case it could be?

Reply:
Yes on Adala side if the Exception for some reason does not have this we will be sending an empty string. Unlikely, but could happen.